source/get-started/cloud-install/qingcloud.rst

```diff
@@ -0,0 +1,245 @@
+      .. figure:: /_figures/qingcloud/QingCloud-6.png
+         :scale: 50 %
+         :alt: 创建SSH密钥
+
+         图6: 创建SSH密钥
+
+      在点击 “创建一个” 按钮之后，页面将跳转到 SSH 密钥创建界面。
+
+   #. 在 SSH 密钥创建界面中，您可以依照图7填写密钥的名称以便记忆，并且选择您需要的加密方法，确认无误后即可点击 **“提交”** 按钮。
+
+      .. figure:: /_figures/qingcloud/QingCloud-7.png
+         :scale: 50 %
+         :alt: 新建SSH密钥
+
+         图7: 新建SSH密钥
+
+      提交之后，将跳出密钥下载按钮。
+
+   #. 出现密钥下载按钮后，请在10分钟之内点击下载按钮完成密钥的下载，并将该密钥妥善保存到本地，以便之后连接主机使用。
+
+      .. figure:: /_figures/qingcloud/QingCloud-8.png
+         :scale: 50 %
+         :alt: 下载SSH密钥
+
+         图8: 下载SSH密钥   
+
+      在关闭下载对话框之后，界面将跳转到之前的 “基本信息设置” 界面
+
+#. 在确保 SSH 密钥已妥善下载保存的情况下，如图9检查主机的基本信息，确认无误后请点击 **“创建”** 按钮。
+
+   .. figure:: /_figures/qingcloud/QingCloud-9.png
+      :scale: 50 %
+      :alt: 确认信息并创建主机
+
+      图9: 确认信息并创建主机
+
+   确认后，QingCloud将会创建 Clear Linux OS 主机，您可以在新的界面中查看当前主机的状态。
+
+
+
+申请公网IP并添加到主机
+################################################
+   
+#. 由于 QingCloud 不会为使用默认网络创建的主机自动分配公网IP地址，所以我们需要手动申请，并添加到主机。如图10点击导航栏左侧的 **“网络与CDN”** 按钮。
+
+   .. figure:: /_figures/qingcloud/QingCloud-10.png
+      :scale: 50 %
+      :alt: 网络与CDN
+
+      图10: 网络与CDN
+
+   点击后，您将来到网络与CDN配置界面。
+
+#. 在新页面中，如图11点击左侧 **“公网IP”** 按钮，并点击中间的 **“申请”** 按钮以进行创建公网IP。
+
+   .. figure:: /_figures/qingcloud/QingCloud-11.png
+      :scale: 50 %
+      :alt: 申请创建公网IP
+
+      图11: 申请创建公网IP
+
+   点击申请后，将跳出提示栏，仔细阅读后按照图12点击 **“继续申请公网IP”** 按钮。
+
+   .. figure:: /_figures/qingcloud/QingCloud-12.png
+      :scale: 50 %
+      :alt: 提示栏确认
+
+      图12: 提示栏确认
+
+   之后将跳转到申请公网IP界面。
+
+#. 在申请公网IP页面中，如图13确认和填写相关信息，包括计费模式和带宽上限（本教程中使用的是流量计费模式并且设置了2Mbps的带宽上限），确认无误后点击 **“提交”** 按钮。
+
+   .. figure:: /_figures/qingcloud/QingCloud-13.png
+      :scale: 50 %
+      :alt: 确认提交公网IP申请
+
+      图13: 确认提交公网IP申请
+
+#. 之后如图14通过导航栏点击 **“计算”**、**“网卡”** 按钮来到网卡界面。
+
+   .. figure:: /_figures/qingcloud/QingCloud-14.png
+      :scale: 50 %
+      :alt: 网卡界面
+
+      图14: 网卡界面
+
+#. 在网卡界面，按照图15选中刚刚创建的 Clear Linux OS 主机的网卡，并点击上方 **“更多操作”** 按钮，再点击 **“绑定公网IPv4”** 按钮。
+
+   .. figure:: /_figures/qingcloud/QingCloud-15.png
+      :scale: 50 %
+      :alt: 绑定选中
+
+      图15: 绑定选中
+
+#. 在绑定公网IP确认界面，按照图16选择刚刚申请完成的公网IP地址，并点击下方 **“提交”** 按钮。 等待片刻后，状态将会变成图17中所示。
+
+   .. figure:: /_figures/qingcloud/QingCloud-16.png
+      :scale: 50 %
+      :alt: 提交绑定
+
+      图16: 提交绑定
+
+   .. figure:: /_figures/qingcloud/QingCloud-17.png
+      :scale: 50 %
+      :alt: 公网IP绑定成功
+
+      图17: 公网IP绑定成功
+
+
+连接到 Clear Linux OS 主机
+################################################
+
+请您点击导航栏左侧 **“计算”**、**“主机”** 按钮，确认当前主机处于正在运行状态，且已绑定了公网IP地址。如图18所示。
+
+.. figure:: /_figures/qingcloud/QingCloud-18.png
+   :scale: 50 %
+   :alt: 确认主机当前处于正常状态
+
+   图18: 确认主机当前处于正常状态
+
+#. 复制当前 Clear Linux OS 主机的公网IP地址，并使用 SSH 客户端进行连接。 这里我们需要用到之前保存的 SSH 密钥。
+#. 在此教程中，以 MobaXterm 客户端为例演示登录过程。请如图19检查各项。用户名我们选择 **root**，密钥请选择之前下载并保存到本地的 SSH 密钥。
+
+   .. figure:: /_figures/qingcloud/QingCloud-19.png
+      :scale: 50 %
+      :alt: SSH 登录主机设置
+
+      图19: SSH 登录主机设置
```

Comment:
the IP address in this screenshot is censored, but not everywhere else. Either display it or censor the IP address in other screenshots too.

Reply:
Thanks for pointing this out. Done!